modules/libESNES/include/esnes/common/types.h

```diff
@@ -0,0 +1,13 @@
+#ifndef ESNES_COMMON_TYPES_H
+#define ESNES_COMMON_TYPES_H
+
+#include <cstdint>
+namespace esnes::common {
```

Comment:
Im not sure how you feel about adding some vertical space between certain things

Reply:
Mis-type, fixed